apps/web/panda.config.ts

```diff
@@ -22,7 +22,7 @@ export default defineConfig({
   ],
 
   // Files to exclude
-  exclude: ['**/*.cy.{js,jsx,ts,tsx}', '**/*/styled-system'],
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** 
```
exclude: ['**/*.spec.{js,jsx,ts,tsx}', '**/*/styled-system'],
```